src/Umbraco.Core/Media/EmbedProviders/Twitter.cs

```diff
@@ -14,7 +14,7 @@ public Twitter(IJsonSerializer jsonSerializer)
 
     public override string ApiEndpoint => "http://publish.twitter.com/oembed";
 
-    public override string[] UrlSchemeRegex => new[] { @"twitter.com/.*/status/.*" };
+    public override string[] UrlSchemeRegex => new[] { @"(?:https?:\/\/)(?:www\.)?(twitter|x)\.com\/.*\/status\/.*" };
```

Comment:
You can make it slightly more compact with:
```(?:https?:\/\/(www\.)?)(twitter|x)\.com\/.*\/status\/.*```
It seems that UrlSchemeRegex is only used to check which embed provider to use, so you can also just use this:
```(https?:\/\/(www\.)?)(twitter|x)\.com\/.*\/status\/.*```

Reply:
I've updated, thanks 💪